components/event-sources/Makefile

```diff
@@ -53,13 +56,16 @@ vendor: Gopkg.lock
 	@echo '+ Pulling vendored dependencies'
 	@dep ensure -v --vendor-only
 
+$(patched_trace): hack/set-span-id.patch vendor
+	@echo '+ Patching opencensus dependency'
+	@git apply ${REPO_ROOT_DIR}/components/event-sources/hack/set-span-id.patch
```

Comment:
You could avoid that complicated path-fu if you just trimmed "components/event-sources/" from the paths inside your patch file:
```diff
--- a/vendor/go.opencensus.io/trace/trace.go
+++ b/vendor/go.opencensus.io/trace/trace.go
```

Reply:
Very good suggestion 👍